config/environment/test.php

```diff
@@ -49,10 +49,9 @@
     }),
 
     'Piwik\Config' => \Piwik\DI::decorate(function ($previous, Container $c) {
-        $testingEnvironment = $c->get('Piwik\Tests\Framework\TestingEnvironmentVariables');
-
         $dontUseTestConfig = $c->get('test.vars.dontUseTestConfig');
         if (!$dontUseTestConfig) {
+            $testingEnvironment = $c->get('Piwik\Tests\Framework\TestingEnvironmentVariables');
```

Comment:
This can be reset, it was done during a change that was reverted, but no harm in keeping this.